ui/constants.py

```diff
@@ -11,3 +11,7 @@
 MARBLE_SIZE = BOARD_CELL_SIZE * 7 / 8
 BOARD_WIDTH = BOARD_CELL_SIZE * BOARD_MAX_COLS
 BOARD_HEIGHT = BOARD_CELL_SIZE * BOARD_MAX_COLS * 7 / 8
+
+DEBUG = True
+DEBUG_LOADS_ON_START = DEBUG
```

Comment:
IMPORTANT: The UI auto-loads `debug.json` when you open the app if it exists. To circumvent this behavior either set this to False or delete `debug.json`